### ISSUE TITLE
acorn-globals is dependent on vulnerable version of acorn

### ISSUE THREAD

**Issue by hjr3** [ForbesLindesay/acorn-globals#53]
Per https://www.npmjs.com/advisories/1488 , acorn v6 is vulnerable. This package (acorn-globals) is used by jsdom. In turn jsdom is used by a lot of other packages, including jest.

I am not familiar with this package, but I will try to investigate what I can do help with an upgrade.

Here is what many people are seeing when they use `npm audit`:

```
                       === npm audit security report ===                        

                                                                                
                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           
                                                                                
                                                                                
  Moderate        Regular Expression Denial of Service                          
  Package         acorn                                                         
  Patched in      >=7.1.1                                                       
  Dependency of   jest [dev]                                                    
                                                                                
  Path            jest > @jest/core > @jest/reporters > jest-runtime >          
                  jest-config > jest-environment-jsdom > jsdom > acorn-globals  
                  > acorn                                                       
                                                                                
  More info       https://npmjs.com/advisories/1488                             
                                                                                
                                                                                
  Moderate        Regular Expression Denial of Service                          
  Package         acorn                                                         
  Patched in      >=7.1.1                                                       
  Dependency of   jest [dev]                                                    

  Path            jest > jest-cli > @jest/core > @jest/reporters >              
                  jest-runtime > jest-config > jest-environment-jsdom > jsdom   
                  > acorn-globals > acorn                                       
                                                                                
  More info       https://npmjs.com/advisories/1488                             
                                                                                
                                                                                
  Moderate        Regular Expression Denial of Service                          
  Package         acorn                                                         
  Patched in      >=7.1.1                                                       
  Dependency of   jest [dev]                                                    
                                                                                
  Path            jest > @jest/core > jest-runner > jest-jasmine2 >             
                  jest-runtime > jest-config > jest-environment-jsdom > jsdom   
                  > acorn-globals > acorn                                       
                                                                                
  More info       https://npmjs.com/advisories/1488                             
                                                                                
                                                                                
  Moderate        Regular Expression Denial of Service                          
  Package         acorn                                                         
  Patched in      >=7.1.1                                                       
  Dependency of   jest [dev]                                                    
                                                                                
  Path            jest > jest-cli > @jest/core > jest-runner > jest-jasmine2 >  
                  jest-runtime > jest-config > jest-environment-jsdom > jsdom   
                  > acorn-globals > acorn                                       
                                                                                
  More info       https://npmjs.com/advisories/1488                             
                                                                                
                                                                                
  Moderate        Regular Expression Denial of Service                          
  Package         acorn                                                         
  Patched in      >=7.1.1                                                       
  Dependency of   jest [dev]                                                    
                                                                                
  Path            jest > @jest/core > jest-runner > jest-runtime > jest-config  
                  > jest-environment-jsdom > jsdom > acorn-globals > acorn      
                                                                                
  More info       https://npmjs.com/advisories/1488                             
                                                                                
  [...snip...]
```

**Comment by hjr3** [comment]
Looks like a bot already opened a PR https://github.com/ForbesLindesay/acorn-globals/pull/50

**Comment by tunnckoCore** [comment]
@ForbesLindesay it's actually pretty huge problem. Because `npm` is doing audit by default, and so all the users using Jest has breaking builds.

I'm doing `yarn audit` as part of the install step and so, I'm affected too.

**Comment by tunnckoCore** [comment]
For anyone having this

```
yarn add EmmaGoodliffe/acorn-globals#patch-1
# or if in monorepo setup
yarn add -W EmmaGoodliffe/acorn-globals#patch-1

# or npm
npm i EmmaGoodliffe/acorn-globals#patch-1
```

**Comment by IdanAdar** [comment]
@TimothyGu New release coming?

**Comment by TimothyGu** [comment]
@IdanAdar Just published 6.0.0.
